### DISCUSSION TITLE
unable to load grapesjs-blocks-basic in angular cli 18

### DISCUSSION THREAD

**Discussion by chetan-bbc** [GrapesJS/grapesjs#6352] [Q&A] [answered]
Hi, please note, we are using latest angular and nodejs version, i want to include basic blocks and am following steps as mentioned in https://github.com/GrapesJS/blocks-basic

after installing `npm i grapesjs-blocks-basic`, i am not able to see the `grapesjs-blocks-basic.min.js` in node_modules folder

![image](https://github.com/user-attachments/assets/de61901d-042f-4080-9327-0c4c954dd479)

if i try using `import grapesjs-blocks-basic`, it is not working either. also i tried using script `"node_modules/grapesjs-blocks-basic/dist/index.js",` it is also not loading the controls.

can you please suggest, what correction is required. Thank you for your help in advance. Thanks.

```
Angular CLI: 18.2.11
Node: 22.9.0
```

**Comment by artf** [comment, marked as answer]
```js
import grapesjs from 'grapesjs';
import plugin from 'grapesjs-blocks-basic';

const editor = grapesjs.init({
  // ...
  plugins: [plugin],
});
```
